tenant/error.go

```diff
@@ -43,6 +43,11 @@ var (
 		Code: influxdb.ENotFound,
 		Msg:  "not found",
 	}
+
+	ErrPasswordTooShort = &influxdb.Error{
```

Comment:
the user service in tenant/service_user.go uses an error defined in error_user.go called `EShortPassword`. You could either use that error or have the user service use this one, I don't care which.